calico_node/glide.yaml

```diff
@@ -3,7 +3,7 @@ import:
 - package: github.com/sirupsen/logrus
   version: ^0.10.0
 - package: github.com/projectcalico/libcalico-go
-  version: v1.7.1
+  version: 6410b9a3ddf7e3b2e33e1e9acd7984e00101ec8f
```

Comment:
should this just be `v1.x-series`?

Reply:
It certainly could be, but that is technically a moving target isn't it (though a very slow one)

Reply:
It's fine to have moving targets in glide.yaml

It's the glide.lock file that will store a specific commit.

Reply:
That said, either a commit or a branch is fine.

Reply:
I feel like the branch is easier to grok when looking at the file. I assume it'll get pinned to a release tag during a `v2.x-series` release anyway.